analyses/MA_cd_2010/doc_MA_cd_2010.md

```diff
@@ -0,0 +1,24 @@
+# 2010 Massachusetts Congressional Districts
+
+## Redistricting requirements
+In Massachusetts, districts must:
+
+1. be contiguous
+2. have equal populations
+3. be geographically compact
+4. preserve county and municipality boundaries as much as possible
+
+
+### Algorithmic Constraints
+We enforce a maximum population deviation of 0.05%. 
+We use a pseudo-county constraint to help preserve county and municipality boundaries.
+
+## Data Sources
+Data for Massachusetts comes from the ALARM Project's [2020 Redistricting Data Files](https://alarm-redist.github.io/posts/2021-08-10-census-2020/).
+
+## Pre-processing Notes
+No manual pre-processing decisions were necessary.
+
+## Simulation Notes
+We sample 5,000 districting plans for Massachusetts over two runs.
+No special techniques were needed to produce the sample.
```

Comment:
can you document the pseudo-county constraint in more detail here?

Something like this
https://github.com/alarm-redist/fifty-states/pull/134/files#diff-f3e497774781ad57e77c839d0560281b322a6fde1df42369fad31e7683bf2366R29-R30

Reply:
Alright, it's fixed! Please let me know if anything else needs changed.

Reply:
Great.  Something seems a bit off here though — Isn't Dukes county really small?   Can you do a `plot(map, pseudo_county)` and show me what that looks like? Makes sense that each muni is its own pseudocounty but unclear to me what's going on with the others.

Reply:
Any update on the muni data here?